runtime/jcl/common/jclcinit.c

```diff
@@ -42,6 +42,7 @@
 #include "j9modron.h"
 #include "omr.h"
 #include "vendor_version.h"
+#include "openj9_version_info.h"
```

Comment:
This will break our (old) b148 builds because openj9_version_info.h doesn't exist in that world.

Reply:
To fix it, see the following example in runtime/util/module.xml (i.e. add something similar to `runtime/jcl/cl_se*/module.xml` where `*` is 7_basic, 9, 10) which allows use of `#if defined(OPENJ9_BUILD)`

```
			<!--Special compilation for openj9_version.c with extra version flags. -->
			<makefilestub data="ifeq ($(OPENJ9_BUILD),true)"/>
			<makefilestub data="openj9_version$(UMA_DOT_O) : CFLAGS += -DOPENJ9_BUILD"/>
			<makefilestub data="endif"/>
```

and

Reply:
Also, since we allow vendor_version.h to override the default values, the include for vendor_version.h should come after openj9_version_info.h